packages/plugin-session-replay-browser/src/session-replay.ts

```diff
@@ -89,7 +92,19 @@ class SessionReplay implements SessionReplayEnrichmentPlugin {
     this.stopRecordingEvents = record({
       emit: (event) => {
         const eventString = JSON.stringify(event);
-        const shouldSplit = shouldSplitEventsList(this.events, eventString, this.maxPersistedEventsSize);
+        // Send the first two recorded events immediately
+        if (this.events.length === 1 && this.currentSequenceId === 0) {
+          this.sendEventsList({
+            events: this.events.concat(eventString),
+            sequenceId: this.currentSequenceId,
+            sessionId: this.config.sessionId as number,
```

Comment:
why is this typed as non-number initially?

Reply:
It's just typed as `number | undefined`, but at this point in the code we know that it's been defined, this allows us to not have to add unnecessary logic to ensure that it is defined (as well as unnecessary tests, the SDK requires 100% test coverage)